Shared/Samples/Style symbols from mobile style file/StyleSymbolsFromMobileStyleFileView.swift

```diff
@@ -0,0 +1,189 @@
+        @Published var currentSymbol: SymbolDetails?
+        
+        /// The current symbol option selections used to create the current symbol.
+        @Published var symbolOptionSelections = SymbolOptions(
+            eyes: "Eyes-crossed",
+            hat: "Hat-cowboy",
+            mouth: "Mouth-frown",
+            color: Color.yellow,
+            size: 40
+        )
+        
+        /// Updates the current symbol with a symbol created from the symbol style using the current option selections.
+        /// - Parameter displayScale: The display scale of the environment for creating the symbol swatch.
+        func updateCurrentSymbol(displayScale: Double) async {
+            // Get the keys from the option selections.
+            var symbolKeys = ["Face1"]
+            symbolKeys.append(contentsOf: symbolOptionSelections.categoryKeys.map { $0.value })
+            
+            // Get the symbol from symbol style using the keys.
+            if let pointSymbol = try? await symbolStyle.symbol(forKeys: symbolKeys) as? MultilayerPointSymbol {
+                // Color lock all layers but the first one.
+                let layers = pointSymbol.symbolLayers
+                for (i, layer) in layers.enumerated() {
+                    layer.colorIsLocked = i != 0 ? true : false
+                }
+                
+                pointSymbol.color = UIColor(symbolOptionSelections.color)
+                pointSymbol.size = symbolOptionSelections.size
+                
+                // Create an image swatch for the symbol using the display scale.
+                if let swatch = try? await pointSymbol.makeSwatch(scale: displayScale) {
+                    // Update the current symbol with the created symbol and swatch.
+                    currentSymbol = SymbolDetails(symbol: pointSymbol, image: swatch)
+                }
+            }
+        }
+    }
+    
+    /// A symbol and its associated information.
+    struct SymbolDetails {
+        /// The symbol from the symbol style.
+        let symbol: Symbol
+        /// The image swatch of the symbol.
+        let image: UIImage
+        /// The name of the symbol as found in the symbol style.
+        let name: String
+        /// The key of the symbol as found in the symbol style.
+        let key: String
+        /// The category of the symbol as found in the symbol style.
+        let category: String
+        /// The human-readable label of the symbol name.
+        var label: String {
+            let splitName = name.replacingOccurrences(of: "-", with: " ").split(separator: " ")
+            return splitName.last?.capitalized ?? name
+        }
+        
+        init(symbol: Symbol, image: UIImage, name: String = "", key: String = "", category: String = "") {
+            self.symbol = symbol
+            self.image = image
+            self.name = name
+            self.key = key
+            self.category = category
+        }
+    }
```

Comment:
You can have the compiler help you here:

```swift
    /// A symbol and its associated information.
    struct SymbolDetails {
        /// The symbol from the symbol style.
        let symbol: Symbol
        /// The image swatch of the symbol.
        let image: UIImage
        /// The name of the symbol as found in the symbol style.
        var name: String = ""
        /// The key of the symbol as found in the symbol style.
        var key: String = ""
        /// The category of the symbol as found in the symbol style.
        var category: String = ""
    }
```

And then add your extension for label:

```swift

extension StyleSymbolsFromMobileStyleFileView.SymbolDetails {
    /// The human-readable label of the symbol name.
    var label: String {
        let splitName = name.replacingOccurrences(of: "-", with: " ").split(separator: " ")
        return splitName.last?.capitalized ?? name
    }
}
```

This has a few benefits:
- using the compiler to synthesize the init
- the struct definition is just the data, the label is not data per se but a function of the data so an extension is appropriate